azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -218,6 +230,13 @@ public void setSubmitTime(long submitTime) {
 
     flowObj.put(SUBMITTIME_PARAM, submitTime);
 
+    List<Map<String, Object>> slaOptions = new ArrayList<>();
+    for (SlaOption slaOption : this.getSlaOptions()) {
+      slaOptions.add(slaOption.toObject());
+    }
+
+    flowObj.put(SLAOPTIONS_PARAM, slaOptions);
```

Comment:
Is this map serialized and saved to the  DB? Could you elaborate how the backward compatibility is handled here and in the change description?